### PR TITLE
update v3relay article from stub to article

### PR DESCRIPTION
**Pull request by paulkania**
squashed version, "builds" on-top of PR [#1001](https://github.com/bitcoinops/bitcoinops.github.io/pull/1001).

### CODE REVIEW

File: _topics/en/version-3-transaction-relay.md [hunk 1]
```diff
@@ -46,20 +46,24 @@ see_also:
...
-## Optional.  Force the display (true) or non-display (false) of stub
-## topic notice.  Default is to display if the page.content is below a
-## threshold word count
-#stub: false
+V3 transaction relay solves rule 3 [transaction pinning][topic version 3 transaction relay]
+and may allow the removal of the CPFP carve-out.
 
-## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
-## Should be less than 500 characters
-excerpt: >
```
Comment (harding) [inline _topics/en/version-3-transaction-relay.md:57]: The excerpt field is required, so you'll need to move the first paragraph of the text back into the excerpt.  (Jekyll automatically generates an excerpt for pages from the first paragraph, but we need the excerpt field to be defined in the YAML in order to perform some additional validation on it, such as ensuring that it is less than 500 characters.)

Comment (paulkania) [reply _topics/en/version-3-transaction-relay.md:57]: Fixed & squashed. git is tricky :)

**Comment by harding** [comment]
@paulkania the current build errors are:

```
_topics/en/version-3-transaction-relay.md:60: MD009 Trailing spaces

_topics/en/version-3-transaction-relay.md:61: MD009 Trailing spaces

_topics/en/version-3-transaction-relay.md:62: MD009 Trailing spaces

_topics/en/version-3-transaction-relay.md:63: MD009 Trailing spaces

_topics/en/version-3-transaction-relay.md:69: MD009 Trailing spaces
```

This means, on the lines indicated (60, 61, 62, 63, and 69), there's an unnecessary space at the end of the line.  We have a test for that because, in some flavors of Markdown, extra spaces at the end of a line are translated into an HTML `<br>` tag.

**Comment by paulkania** [comment]
I hate how unnecessarily complex I've made this PR...my apologies.
I tangled myself up in rebasing and merging conflicts, and decided to try to nuke my local repo and start fresh.
The issue now is that I'm unable to push a new PR or update this one : requested url error 403.

I promise to re-visit the git-documentation asap.

**Comment by Xekyo** [comment]
@paulkania: If you tell us what specific problems you are running into, we can probably walk you through. Alternatively, one of us can squash your PR for you, and you can continue from there.

**Comment by paulkania** [comment]
@Xekyo I blundered with my branching and merging. I re-opened this PR as [#1009 ](https://github.com/bitcoinops/bitcoinops.github.io/pull/1009)

**Pull request closed without merging**